RRTPlannerROS: {
  goal_tolerance: 0.20,

  rrt: {
    step: 0.10,
    min_num_nodes: 500,
    max_num_nodes: 5000,
  }
}
